sunpy/image/tests/test_coalignment.py

```diff
@@ -52,10 +52,10 @@ def aia171_test_template(aia171_test_shift,
                          aia171_test_map_layer,
                          aia171_test_map_layer_shape):
     # Test template
-    a1 = aia171_test_shift[0] + aia171_test_map_layer_shape[0] / 4
-    a2 = aia171_test_shift[0] + 3 * aia171_test_map_layer_shape[0] / 4
-    b1 = aia171_test_shift[1] + aia171_test_map_layer_shape[1] / 4
-    b2 = aia171_test_shift[1] + 3 * aia171_test_map_layer_shape[1] / 4
+    a1 = int(aia171_test_shift[0] + aia171_test_map_layer_shape[0] / 4)
+    a2 = int(aia171_test_shift[0] + 3 * aia171_test_map_layer_shape[0] / 4)
+    b1 = int(aia171_test_shift[1] + aia171_test_map_layer_shape[1] / 4)
+    b2 = int(aia171_test_shift[1] + 3 * aia171_test_map_layer_shape[1] / 4)
```

Comment:
I would suggest to make just integer what we want to be integer...
`aia171_test_shift` should be integer already, and so the 3.  Is it the division? should we use `//` instead?
I say so because we could get into something weird (maybe not in this example) by setting it all integer. But if we just change what it need to be changed it could be easier to debug in the future.